e2e/manifests/overlays/daemonset-lvmd/node.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: topolvm-system
+  name: node
+spec:
+  template:
+    spec:
+      containers:
+        - name: topolvm-node
+          image: topolvm:dev
+          imagePullPolicy: Never
+          volumeMounts:
+            - name: device-dir
+              mountPath: /dev
+
+        - name: csi-registrar
+          image: topolvm:dev
+          imagePullPolicy: Never
+
+        - name: liveness-probe
+          image: topolvm:dev
+          imagePullPolicy: Never
+
+      volumes:
+        - name: lvmd-socket-dir
+          hostPath:
+            path: /tmp/topolvm/lvmd_lvmd
```

Comment:
This can be replaced by kustomize patch. so it is not necessary for entire content coping.